### ISSUE TITLE
ma.masked does not have fill_value attribute

### ISSUE THREAD

**Issue by abalkin** [numpy/numpy#4595]
```
>>> np.ma.masked.fill_value
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "numpy/ma/core.py", line 3353, in get_fill_value
    if self._fill_value is None:
AttributeError: 'MaskedConstant' object has no attribute '_fill_value'
```

and as a consequence

```
>>> np.ma.masked.filled()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "numpy/ma/core.py", line 3442, in filled
    fill_value = self.fill_value
  File "numpy/ma/core.py", line 3353, in get_fill_value
    if self._fill_value is None:
AttributeError: 'MaskedConstant' object has no attribute '_fill_value'
```

**Comment by twiecki** [comment]
Just ran into the same issue.

**Comment by eric-wieser** [comment]
This is fixed in 1.14-dev, and perhaps earlier:
```
>>> np.ma.masked.fill_value
1e+20
```
